### ISSUE TITLE
Unable to use with antfu's eslint-config

### ISSUE THREAD

**Issue by RohitKaushal7** [francoismassart/eslint-plugin-tailwindcss#393]
Hey I'm trying to use the plugin with antfu's eslint-config like this.. 

```js
// eslint.config.js
import antfu from '@antfu/eslint-config'
import js from '@eslint/js'
import tanstackQuery from '@tanstack/eslint-plugin-query'
import reactHooks from 'eslint-plugin-react-hooks'
import reactRefresh from 'eslint-plugin-react-refresh'
import tailwind from 'eslint-plugin-tailwindcss'
import globals from 'globals'
import tseslint from 'typescript-eslint'

const config = tseslint.config(
  { ignores: ['dist'] },
  {
    extends: [
      js.configs.recommended,
      ...tseslint.configs.recommended,
    ],
    files: ['**/*.{ts,tsx}'],
    languageOptions: {
      ecmaVersion: 2020,
      globals: globals.browser,
    },
    plugins: {
      'react-hooks': reactHooks,
      'react-refresh': reactRefresh,
      '@tanstack/query': tanstackQuery,
      'tailwindcss': tailwind,
    },
    rules: {
      ...reactHooks.configs.recommended.rules,
      'react-refresh/only-export-components': [
        'warn',
        { allowConstantExport: true },
      ],
    },
  },
)

export default antfu({
  ...config,
})
```

But I am not getiing the classname sorting and other features of this plugin and eslint gives this error

```
Uncaught exception received.
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './resolveConfig' is not defined by "exports" in /home/.../node_modules/tailwindcss/package.json
    at exportsNotFound (node:internal/modules/esm/resolve:304:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:651:9)
    at resolveExports (node:internal/modules/cjs/loader:634:36)
    at Module._findPath (node:internal/modules/cjs/loader:724:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1211:27)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/home/rohit/.../node_modules/eslint-plugin-tailwindcss/lib/util/customConfig.js:5:23)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
```

I also tried to put the config in the extends section as
`...tailwind.configs['flat/reccomended']` but that errors too.

what is the right way to use this plugin? as the documentation only shows the usage as this independent plugin.

**Comment by withxat** [comment]
This type of error is typically caused by `tailwindcss@^4` because `eslint-plugin-tailwindcss` does not currently support it. Using `tailwindcss@^3` should resolve the issue.

**Comment by withxat** [comment]
https://github.com/francoismassart/eslint-plugin-tailwindcss/issues/325#issuecomment-2499287600

**Comment by LouisMazel** [comment]
You have to use it like this : 

```ts
import antfu from '@antfu/eslint-config'
import tailwind from 'eslint-plugin-tailwindcss'

export default antfu(
  {
    formatters: true,
    vue: true,
    rules: {
       ...
    },
  },
  ...tailwind.configs['flat/recommended'],
)

```
